### PR TITLE
feat: [CHAOS-2619]: Adds cf-app-stop fault docs

### PR DESCRIPTION
**Pull request by neelanjan00**
# Harness Developer Pull Request
Adds cf-app-stop fault docs

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    <th> Description </th>
+    <th> Notes </th>
+  </tr>
+  <tr>
+    <td> faultInjectorPort </td>
+    <td> The local server port to be used by the fault-injector utility </td>
+    <td> Default: <code>50320</code>. If the default port is unavailable, a random port will be selected in the range of <code>50320-51320</code> </td>
+  </tr>
+  <tr>
+    <td> skipSSLValidation </td>
```
Comment (ispeakc0de) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md:57]: Can we add duration field?

**Review by ispeakc0de — APPROVED** [review]

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+id: cf-app-stop
+title: CF app stop
+---
+CF app stop fault stops a Cloud Foundry app and later starts it.
+
+![CF App Stop](./static/images/cf-app-stop.png)
+
+## Use cases
+CF app stop fault:
+- Checks app resilience against abrupt stopping.
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md:11]:
```suggestion
- Checks resilience against abrupt of app components/microservices.
```

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+- Validates the effectiveness of disaster recovery and high availability of the app.
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md:12]:
```suggestion
- Validates the effectiveness of disaster recovery and high availability of the application.
```

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+<h3>Mandatory fields</h3>
+<table>
+  <tr>
+    <th> Tunable </th>
+    <th> Description </th>
+    <th> Notes </th>
+  </tr>
+  <tr>
+    <td> cfDeploymentPlatform </td>
+    <td> Deployment platform used for cloud foundry, with respect to where the infrastructure is executing </td>
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md:24]:
```suggestion
    <td> Deployment platform used for cloud foundry, with respect to where the infrastructure is hosted </td>
```

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+  </tr>
+  <tr>
+    <td> faultInjectorPort </td>
+    <td> Local server port used by the fault-injector utility </td>
+    <td> Default: <code>50320</code>. If the default port is unavailable, a random port in the range of <code>50320-51320</code> is selected. </td>
+  </tr>
+  <tr>
+    <td> duration </td>
+    <td> Duration through which chaos is injected into the target resource (in seconds). </td>
+    <td> Default: 30 s. </td>
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cf-app-stop.md:59]:
```suggestion
    <td> Default: 30 </td>
```
As the initial cut still doesn't support units

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+<!-- Import statement for Custom Components -->
+
+import FaultDetailsCard from "@site/src/components/ChaosEngineering/FaultDetailsCard";
+import ExperimentListSection from "@site/src/components/ChaosEngineering/ExperimentListSection"
+import { experiments } from "./experiments"
+
+<!-- Heading Description -->
+## Introduction
+
+Cloud Foundry faults disrupt the Cloud Foundry resources. This deteriorates the performance of the app for the duration of the chaos experiment.
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md:15]:
```suggestion
Cloud Foundry faults disrupt the functioning of Cloud Foundry resources. This deteriorates the performance of the app for the duration of the chaos experiment.
```

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+<!-- Experiment List and Search Bar (every experiment added below, need to be added in this file also) -->
+
+<ExperimentListSection experiments={experiments} />
+
+<FaultDetailsCard category="cloud-foundry">
+
+### CF app stop
+
+CF app stop injects app stop chaos for a Cloud Foundry app.
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md:25]:
```suggestion
CF app stop stops a Cloud Foundry application for a specified duration.
```

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+- Checks app resilience against abrupt stopping.
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md:27]:
```suggestion
- Checks resilience against abrupt stop of application components/microservices.
```

File: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+- Validates the effectiveness of disaster recovery and high availability of the app.
+
+<accordion color="green">
+<summary>Use cases</summary>
+
+- Checks app resilience against abrupt stopping.
```
Comment (ksatchit) [inline docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md:33]:
```suggestion
- Checks resilience against abrupt stop of application components/microservices..
```

**Pull request merged**